core/src/main/scala/spinal/core/sim/package.scala

```diff
@@ -356,50 +358,49 @@ package object sim {
     }
   }
 
-  /**
-    * Add implicit function to Bits
-    */
-  implicit class SimBitsPimper(bt: Bits) {
+  protected abstract class RandomizableBitVector(bt: BitVector, longLimit: Int) {
```

Comment:
Hmm how does `protected` behave in a package? Is it like `private`?

Reply:
changed to private, don't know why I used protected...
The difference, after looking it up: protected classes can be returned by functions to outside of the package, private can not

Reply:
Edit: changed by to protected, if the class is private then it is not allowed to escape the context at all, so a public class can´t derive from it ... the blog where I looked it up was a bit...wrong